codegen/kuberesource/crd_test.go

```diff
@@ -40,9 +40,9 @@ var _ = Describe("Crd", func() {
 			o, err := CustomResourceDefinitions(grp)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(o).To(HaveLen(1))
-			// note: we intentionally provide the "5770277ee8c9bc92" hash in the test, as it shouldn't change
+			// note: we intentionally provide the "b6ec737002f7d02e" hash in the test, as it shouldn't change
```

Comment:
do you know why this hash (and all the hashes in the CRDs) changed?

Reply:
Not entirely sure, I can try and track down why. 
My initial thought is potentially because of related go dependency upgrades on packages that are used to generate the CRDs. Particularly `code-generator`, but that is a complete guess rn.